vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManager.java

```diff
@@ -267,6 +332,11 @@ public void deleteChoiceBlockById(String blockId, String slideId) throws BlockDo
     public void updateTextBlock(TextBlock textBlock) {
         textBlockRepo.save((TextBlock) textBlock);
     }
+    
+    @Override
+    public void updateSpaceBlock(SpaceBlock spaceBlock) {
```

Comment:
let's call this `saveSpaceBlock` (since I just talked to Avirup about something like this).